settings.gradle

```diff
@@ -1,3 +1,5 @@
 rootProject.name = 'core'
 include 'kgraphql'
 include 'kgraphql-ktor'
+include 'example'
```

Comment:
could you rename this to `kgraphql-example`, just to follow the same pattern?

Reply:
Done 👍